### DISCUSSION TITLE
project xtended sync error

### DISCUSSION THREAD

**Discussion by alecchangod** [ROM-builders/temporary#20146] [General]
https://cirrus-ci.com/github/ROM-builders/temporary/pull/20045
i keep getting different kind of sync error here, can't make a successful sync and build fine. I have tested the local_manifest and all are fine but sometime it didn't sync some repo on cirrus ci and tell me the file was missing.
https://cirrus-ci.com/task/6199488813989888?logs=build#L6

local_manifest https://github.com/alecchangod/local_manifest/blob/anx-xtended/monet.xml

I am waiting for this build to make a release. Thanks.

**Comment by Apon77** [comment]
Clang trb may give libc version error

**Reply by alecchangod** [reply]
i have reverted it to old version and it work fine. now the problem was repo syncing
